### DISCUSSION TITLE
How to enable plugins by env variables?

### DISCUSSION THREAD

**Discussion by delfer** [lobehub/lobehub#3188] [Self Hosting]
Hello!
i want to enable some plugins on redeploy.
```
PLUGIN_SETTINGS: getWebsiteContent,searchGoogle
```
Doesn't helps(

**Comment by arvinxx** [comment]
Please check here: https://lobehub.com/docs/self-hosting/environment-variables/basic#default-agent-config

**Comment by delfer** [comment]
@arvinxx It's not exactly what I wants. I want to install plugins on deploy
<img width="489" alt="image" src="https://github.com/lobehub/lobe-chat/assets/3233181/eb961b62-1504-4f45-8aa8-9e9c3ded7e63">

**Reply by arvinxx** [reply]
because `getWebsiteContent` and `searchGoogle` are not valid plugins id
